src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+@import "./variables";
+@import "./utils/mixins";
+
 body {
   margin: 0;
+  font-family: $font-family;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 100vh;
+
+  margin: auto;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  padding: $padding-box;
+
+  width: $day-width * 7 + $gap * 6;
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $num in $days-of-week {
+    &--start-#{$day} .calendar__day:first-child {
+      margin-left: ($day-width + $gap) * ($num);
+    }
+  }
+
+  &__day {
+    @include calendar-day;
+
+    &:hover,
+    &--today {
+      background-color: $color-pink;
+    }
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+
+    @for $i from 1 through $month-length {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+        font-size: 30px;
```

Comment:
Font size is the same, so you don't need to add this property in a loop, do it just once